app/lib/mini-course.cjsx

```diff
@@ -17,81 +16,73 @@ module.exports = React.createClass
 
   statics:
     find: ({workflow}) ->
-      # Prefer fetching the tutorial for the workflow, if a workflow is given.
+      # Prefer fetching the tutorial for the workflow, so we know which one to fetch if multiple exist.
       if workflow?
         apiClient.type('tutorials').get workflow_id: workflow.id, kind: "mini-course"
           .then ([minicourse]) ->
             minicourse
```

Comment:
Technically mini-courses are just a tutorial resource with `kind: 'mini-course'` set and we allow multiple tutorials (in the back end) per workflow. However, in the project builder interface, we are only allowing one tutorial and/or one mini-course per workflow to be linked. With this limitation in the builder UI, I can be assured it's the first tutorial that is mini-course kind that is returned.